charts/rancher-istio-1.9/rancher-istio/1.9.800/Chart.yaml

```diff
@@ -0,0 +1,21 @@
+annotations:
+  catalog.cattle.io/auto-install: rancher-kiali-server-crd=1.32.100
```

Comment:
Here we still use the rancher-kiali-server-crd chart 1.32.100 which already existing in the repo. 

The new charts `rancher-kiali-server-1.32.100+up1.32.0` and `rancher-kiali-server-1.35.100+up1.35.0` will be removed in future.